ci/do_ci.sh

```diff
@@ -85,21 +94,22 @@ if [[ "$1" == "bazel.release" ]]; then
   setup_clang_toolchain
   echo "bazel release build with tests..."
   bazel_binary_build release
+  bazel_release_options="test ${BAZEL_TEST_OPTIONS} -c opt ${MEMORY_TEST_EXACT_ARGS}"
```

Comment:
Please stick to all caps vars for consistency with the rest of the scripts.

Reply:
done